### ISSUE TITLE
[css-images] Specific algorithms for image-rendering

### ISSUE THREAD

**Issue by progers** [w3c/csswg-drafts#6252]
The [`image-rendering`](https://drafts.csswg.org/css-images-3/#the-image-rendering) property supports values such as `smooth` and `high-quality` but these are not specific enough. Bilinear is preferable in some cases ([example](https://blog.codinghorror.com/the-myth-of-infinite-detail-bilinear-vs-bicubic/)), but not others, and we get bugs about this in chrome ([crbug.com/562162](https://crbug.com/562162)). To change our defaults to be cubic in more cases, we need a way for users to request other algorithms.

Could we support the following new values for `image-rendering`?
* `nearest-neighbor`
* `bilinear`
* `trilinear` (using mipmaps)
* `cubic-mitchell`

The first three of these are ubiquitous and are typically available in hardware. An alternative to `cubic-mitchell` is to just specify `cubic` and leave the details up to implementations (see [this page](https://entropymine.com/imageworsener/bicubic/)).

**Comment by svgeesus** [comment]
I agree that more user control is desirable. [`sinc` aka `Lanczos` resampling](https://renderpool.net/blog/image-scaling/) should also be considered.

I also agree that bicubic catmull-rom is the more generally applicable of the various cubic options.

Lastly, an opt-in to specify resampling in linear-light rather than gamma-encoded space is desirable. The demo page you linked for bicubic casually mentions this was done but does not show the result of _not_ doing it.

**Comment by tabatkins** [comment]
Currently the values are intentionally vague because implementations didn't want to require specific algorithms, both because it would potentially require more implementation work they didn't think was very necessary, and because they wanted the freedom to upgrade and downgrade their scaling algorithms as needed to accommodate the page and device load.

I'm happy to add more specific algorithm requirements (possibly behind specifically-named keywords), but I'd want some impl support beyond Chrome. ^_^
